content/en/llm_observability/setup/sdk/python.md

```diff
@@ -519,45 +519,21 @@ def rag_workflow(user_question):
 
 ## Evaluations
 
-The LLM Observability SDK provides the methods `LLMObs.export_span()` and `LLMObs.submit_evaluation()` to help your traced LLM application submit evaluations to LLM Observability.
```

Comment:
we probably wanna keep a reference to the old function and mention that it's deprecated so that if someone tries to go back to our docs to find it they aren't too confused.